spec/acceptance/nodesets/archlinux-2-x64.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Should we keep this and the ec2 nodesets out? I've been wondering how useful they are.